tests/conftest.py

```diff
@@ -230,3 +230,30 @@ def clean_flask_app_config(app):
         stock_config = app.config.copy()
         yield
         app.config = stock_config.copy()
+
+
+@pytest.fixture(autouse=True)
+def use_cookie(client, default_guard):
+    class withCookie:
```

Comment:
🔥

Reply:
Either you're telling me that there is somethign wrong or that this part is just lit. I'll gladly take the last one 😅

Reply:
Yeah, it's the latter. I like it